core/server/permissions/effective.js

```diff
@@ -7,6 +8,10 @@ effective = {
     user: function (id) {
         return Models.User.findOne({id: id, status: 'all'}, {include: ['permissions', 'roles', 'roles.permissions']})
             .then(function (foundUser) {
+                if (!foundUser) {
+                    return Promise.reject(new errors.NotFoundError(id));
```

Comment:
This error should probably use https://github.com/TryGhost/Ghost/blob/4571a40f2ae7adff9b016b274a9e4b910bc82260/core/server/translations/en.json#L244-L244?